scripts/getStatus.js

```diff
@@ -111,12 +112,23 @@ async function runScript() {
 		updateResult(swagger, validationErrors, true);
 	  }
   }
-  
+  console.log(`\t- Running Linter.`);
+
   for (let readme of readmesToProcess) {
-	  console.log(`Configuration file: "${readme}"`);
-      const linterErrors = await runLinter(readme);
+	  console.log(`Linter Validation on configuration file: "${readme}"`);
+      let linterCmd = 'autorest ' + readme + ' --azure-validator=true --Validation --message-format=json';
+      const linterErrors = await runCmd(linterCmd);
       updateResult(readme, linterErrors, true);
   }
+  console.log(`\t- Running Model Validator.`);
+
+    //model validator run
+  for (let readme of readmesToProcess) {
+      console.log(`Model Validation on configuration file: "${readme}"`);
+      let modelValCmd = 'autorest --version=2.0.4174 --model-validator --message-format=json ' + readme;
```

Comment:
To get this pulling data now, I assume we still need to target that version.  If we don't, let me know and I can remove the version requirement.  In all other jobs we run as latest so if it is ready, I can remove that.

Reply:
at this point you need at least this version, I'm pointing it out though, because you will want to update this when autorest releases next, if not, this command will get "stuck" in a version.